src/ws-date-picker/ws-date-picker.js

```diff
@@ -67,7 +68,8 @@ export class WSDatePicker extends Component {
       onChange: this.onChange.bind(this)
     });
     // Prevent default change event bubbling
-    this.input.addEventListener('change', event => event.stopPropagation(), true);
+    this.input.addEventListener('change', this.stopPropagation);
```

Comment:
was there a bug or need for updating this ?

Reply:
The reason was simply a memory leak. Elements which still are referenced like with a event listener will not be cleaned up by the garbage collector. If you do addEventListener with a arrow or anonymus function you can not remove the listener by just applying this arrow function again as this is a new instance with a new identifier. Therefore I moved this into the class so that I can reference the exactly same function for adding and removing the listener